tests/product_tests/sunbeam/juju_tests.py

```diff
@@ -0,0 +1,88 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos_tests import StageOneReportTest
+
+
+class JujuBasicTest(StageOneReportTest):
+    """Ensure that a basic execution runs as expected with simple deployment.
+
+    :avocado: tags=sunbeam
+    """
+
+    sos_cmd = '-v -o juju'
+    arch = ['x86_64']
+
+    ubuntu_only = True
+
+    def test_unit_agent_conf_collected(self):
+        self.assertFileCollected('/var/lib/juju/agents/machine-0/agent.conf')
+        self.assertFileCollected('/var/log/juju/machine-0.log')
+
+#    def test_unit_commands_collected(self):
+#        cmds_to_check = [
+#            'juju_engine_report',
+#            'juju_goroutines',
+#            'juju_heap_profile',
+#            'juju_leases',
+#            'juju_metrics',
+#            'juju_pubsub_report',
+#            'juju_presence_report',
+#            'juju_statepool_report',
+#            'juju_statetracker_report',
+#            'juju_unit_status',
+#        ]
+#
+#        for the_cmd in cmds_to_check:
+#            self.assertFileCollected(f'sos_commands/juju/{the_cmd}')
```

Comment:
I have left this here, as we need to add this to functional tests. We need to now figure out what happened in my previous PR on adding these commands, as we should be capturing this.

Once that is resolved, we can enable this section.